webpack.config.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+const webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = (env, argv) => {
+  const { mode } = argv;
+
+  return {
+    devtool: mode === 'production'
+      ? 'source-map'
+      : 'cheap-module-eval-source-map',
+    entry: {
+      'fieldmanager': [
+        './js/fieldmanager.js',
+        './css/fieldmanager.css',
+        './css/fieldmanager-options.css',
+      ],
+      'fieldmanager-autocomplete': './js/fieldmanager-autocomplete.js',
+      'fieldmanager-colorpicker': './js/fieldmanager-colorpicker.js',
+      'fieldmanager-datepicker': './js/fieldmanager-datepicker.js',
+      'fieldmanager-draggablepost': [
+        './js/fieldmanager-draggablepost.js',
+        './css/fieldmanager-draggablepost.css',
+      ],
+      'fieldmanager-group-tabs': [
+        './js/fieldmanager-group-tabs.js',
+        './css/fieldmanager-group-tabs.css',
+      ],
+      'fieldmanager-media': './js/media/fieldmanager-media.js',
+      'fieldmanager-select': './js/fieldmanager-select.js',
+      'fieldmanager-quickedit': './js/fieldmanager-quickedit.js',
+      'fieldmanager-options': './css/fieldmanager-options.css',
+      'richtext': './js/richtext.js',
```

Comment:
Could these files be organized into module/entry-specific directories? Doing so might make the bundle structure more intuitive at a glance, as the file/directory structure could indicate which files are bundled together.

I know we may want to retain the `fieldmanager-*` naming convention for the built assets, but now that we're compiling them via Webpack, perhaps the `fieldmanager-` prefix could be dropped from the source files?

Reply:
@montchr Yes, I considered this as well. The intention was to keep the changes minimal at best. But considering some of your findings here it may best serve to restructure for organization purposes. Additionally, if we were to Sassify the CSS the restructuring would benefit those efforts